packages/react/src/components/AccountSettings/ChangePassword/defaults.tsx

```diff
@@ -22,7 +24,7 @@ const DEFAULTS: Required<ChangePasswordComponents> = {
   CurrentPasswordField: DefaultPasswordField,
   NewPasswordField: DefaultPasswordField,
   ConfirmPasswordField: DefaultPasswordField,
-  SubmitButton: Button,
+  SubmitButton: (props) => <Button {...props} />,
```

Comment:
This fixes error:

```
Type 'WeakValidationMap<Pick<PrimitivePropsWithRef<ButtonProps, "button">, "area" | "slot" | "title" | "id" | "className" | "testId" | "alignSelf" | "backgroundColor" | ... 344 more ... | "onTransitionEndCapture"> & RefAttributes<...>>' is not assignable to type 'WeakValidationMap<Partial<ButtonPrimitiveProps> & Required<Pick<ButtonPrimitiveProps, "isDisabled">>>'.
  Types of property 'isDisabled' are incompatible.
    Type 'Validator<boolean | null | undefined> | undefined' is not assignable to type 'Validator<boolean> | undefined'.
       Type 'Validator<boolean | null | undefined>' is not assignable to type 'Validator<boolean>'.
         Type 'boolean | null | undefined' is not assignable to type 'boolean'.
           Type 'undefined' is not assignable to type 'boolean'.
```

which was complaining that `isDisabled` is required on override component type, but is optional on `Button` itself.